### DISCUSSION TITLE
Best personal information manager that integrates with SingleFile? (Obsidian, etc.)

### DISCUSSION THREAD

**Discussion by silversheaf** [gildas-lormeau/SingleFile#1424] [General]
I'm wondering what knowledge management software (Obsidian, Joplin, Tana, Craft, etc.) best integrates with SingleFile. Is there anything that SingleFile can send to it with one click, makes the result fully searchable, and preserves metadata like date retrieved and URL?

**Comment by reorx** [comment]
I'm also wondering if it is possible to browse a SingleFile archive in Obsidian. It seems this could be achieved by writing a plugin.

**Reply by gildas-lormeau** [reply]
Did you try https://github.com/nuthrash/obsidian-html-plugin?

**Comment by reorx** [comment]
Just know it, and it works pretty well, thanks for telling me!

Sincerely,
Xiao Meng
